README.markdown

```diff
@@ -752,7 +752,7 @@ ts-protoc --ts_proto_out=./output -I=./protos ./protoc/*.proto
 # Todo
 
 - Support the string-based encoding of duration in `fromJSON`/`toJSON`
-- Make `oneof=unions` the default behavior in 2.0
+- Make `oneof=unions-value` the default behavior in 2.0
```

Comment:
Ah nice! I definitely agree this `value` field is better and should be the default.

We're in luck that a 2.x release is likely going to happen in the next ~month or so, via #1058, so we should plan on flipping this default over, and honestly probably just removing the option/old behavior all together.

Reply:
Sounds great, feel free to ping me if you want help with that.

Reply:
Great! I'll ping you once Timo's PR lands as an 2.0.0 alpha and would be great to get this flipped over.